apps/designer/app/canvas/canvas.tsx

```diff
@@ -55,7 +55,7 @@ const useElementsTree = () => {
       if (rootInstance === undefined) return;
 
       const { instanceId, updates } = change;
-      setInstanceChildrenMutable(instanceId, updates, rootInstance);
+      utils.tree.setInstanceChildrenMutable(instanceId, updates, rootInstance);
```

Comment:
This is not optimizable by bundler. We will get `a.tree.setInstanceChildrenMutable` across whole bundle

Reply:
yeah, that's expected, we moved it there long time ago, alternative would be to export it as treeUtils so we can import treeUtils and not utils, right?

Reply:
this PR is touching it because somebody brought back to the app files we moved to project quite some time ago

Reply:
we don't expect these project stuff to be tree shaken since it's all in the designer, but maybe we are wrong by ignoring it in general and setting a bad example